src/types/common.d.ts

```diff
@@ -104,6 +104,14 @@ export interface RenderContext {
   measureText(text: string): { width: number };
 }
 
+export interface RenderTieParams {
```

Comment:
On second thought, let's just inline this :-) I prefer the duplication to this very specific interface.

Reply:
done :)